app/tasks.py

```diff
@@ -379,3 +380,65 @@ def make_query(self):
     def update_counter(self, counter, person):
         person.update_index(['old', 'new'])
         person.put()
+
+
+class NotifyBadReviewStatus(utils.BaseHandler):
+    """This task sends email notification when the number of unreviewed notes
+    exceeds threshold.
+    """
+    ACTION = 'tasks/notify_bad_review_status'
+
+    def task_name(self):
+        return 'notify-bad-review-status'
+
+    def get(self):
+        if self.repo:
+            try:
+                count_of_unreviewed_notes = \
+                    model.Note.unreviewed_record_count(self.repo)
+            except runtime.DeadlineExceededError:
+                logging.info("DeadlineExceededError occurs")
+                self.add_task_for_repo(
+                    self.repo, self.task_name(), self.ACTION)
+            except datastore_errors.Timeout:
+                logging.info("Timeout occurs in datastore")
+                self.add_task_for_repo(
+                    self.repo, self.task_name(), self.ACTION)
+            finally:
+                self._notify(count_of_unreviewed_notes)
+
+    def _notify(self, count_of_unreviewed_notes):
+        # TODO: Response should be modified
+        if self._is_ok_to_notify(count_of_unreviewed_notes):
+            self.send_mail(self._get_admin_user_email(),
+                           subject="PersonFinder: Please review your notes",
+                           body="{0} notes are not reviewed".format(
+                               count_of_unreviewed_notes))
+            return True
+        else:
+            return False
+
+    def _is_ok_to_notify(self, count_of_unreviewed_notes):
+        email = self._get_admin_user_email()
+        if not email:
+            return False
+
+        thres = self._get_thres_unreviewed_notes()
+        if count_of_unreviewed_notes >= thres:
+            return True
+        else:
+            return False
+
+    def _get_admin_user_email(self):
+        return self.config.get('admin_user_email')
+
+    def _get_thres_unreviewed_notes(self):
+        try:
+            return int(self.config.get('thres_unreviewed_notes'))
```

Comment:
Currently, I limit range of input for 'thres_unreviewed_notes' by <input type="number" min="0" name="unreviewed_notes_email_threshold" id="unreviewed_notes_email_threshold" size=24 value="{{view_config.unreviewed_notes_email_threshold}}">. Also, I set default value by const.DEFAULT_UNREVIEWED_NOTES_EMAIL_THRESHOLD. Thus now I think I don't need error handling for this. Do you think it is better to add assert statement to fail the request?